profiles/base.yaml

```diff
@@ -123,6 +123,10 @@ offensive_spells:
 # offensive_spell_cycle is a list of offensive_spells names in the order you want to cast them.  Most useful if you have multiple TM or DEBIL spells to cast
 offensive_spell_cycle:
 # list of actions to take while aiming when stealth still needs to be trained
+
+# toggle to not release cyclic spells at the end of combat.  USE WITH CAUTION as most combat cyclics will get you arrested in a hurry.
+do_not_release_combat_cyclic: false
```

Comment:
@Hiinky I often get confused with the double negatives with toggles/flags. Thoughts on renaming this to `release_combat_cyclic` where `true` means yes, release it after combat, and `false` means no, don't release it.

Then `DRCA.release_cyclics unless get_settings.do_not_release_combat_cyclic` would become `DRCA.release_cyclics if get_settings.release_combat_cyclic`

Reply:
I also like this idea